Automattic/pocket-casts-android
# Update bookmark visibility on filter episodes list

## Description

This PR updates the episodes in the Filter section(In Progress and New Releases) with the bookmark changes.
The Bookmark Icon is now visible when we add bookmarks and delete bookmarks. 

Fixes #1310 
## Testing Instructions
1. Login to a paid account
2. Sign up at Patron and toogle Bookmarks(Settings, Beta -features)
3.  Add Podcasts
4. Play some episodes
5. Go to filters
6. Click New Releases/ In Progress
7. Play Episode from new releases/ In progress
8. Add Bookmark
9. Go back to the episodes list and check the bookmark icon is now visible 
10. Delete the bookmark, Observe the Icon is now deleted

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/95022986/1dce49f3-55bf-4c04-abad-5882341bcb57



## Checklist
 
#### I have tested any UI changes...
